### ISSUE TITLE
Resume training model with other learning rate

### ISSUE THREAD

**Issue by ChrisAlthaus** [facebookresearch/detectron2#2312]
Hi,

at the moment I'm resuming a trained model checkpoint by calling:
```
trainer = DefaultTrainer(cfg)
trainer.resume_or_load()  # load last checkpoint or MODEL.WEIGHTS
trainer.train()
```
As the documentation says: "... Resuming means loading all available states (eg. optimizer and scheduler)", the learning rate will be the same as those of the checkpoint.  [https://detectron2.readthedocs.io/modules/engine.html#detectron2.engine.defaults.DefaultTrainer.resume_or_load](url)
But i want to resume the training with the learning rate defined by CFG.SOLVER.BASE_LR with corresponding GAMMA. How can I achieve this? Initialize a new lr scheduler, but if yes then where do i need to modify the source code?

Greetings,
Christian

**Comment by ppwwyyxx** [comment]
According to https://detectron2.readthedocs.io/modules/engine.html#detectron2.engine.defaults.DefaultTrainer.resume_or_load you just need to set `resume=False` so learning rate will not be resumed.

More generally, you can use `resume=True` but edit the checkpoint with `torch.load/save` to decide what to resume.
